lib/internal/streams/utils.js

```diff
@@ -4,8 +4,17 @@ const {
   Symbol,
   SymbolAsyncIterator,
   SymbolIterator,
+  TypedArrayPrototypeGetBuffer,
+  TypedArrayPrototypeGetByteOffset,
+  TypedArrayPrototypeGetByteLength,
 } = primordials;
 
+const { normalizeEncoding } = require('internal/util');
+const { FastBuffer } = require('internal/buffer');
+const { TextEncoder } = require('internal/encoding');
+
+const encoder = new TextEncoder();
```

Comment:
Should we do the instantiation is a lazy way?

Reply:
`TextEncoder` instance does not hold any reference. If your concern is startup time, there's no need.